tests/testthat/test-TealDatasetConnector.R

```diff
@@ -564,7 +564,7 @@ testthat::test_that("code_dataset_connector - library calls", {
     label = "ADRS dataset"
   )
 
-  data <- cdisc_data(adsl, adtte, adrs, check = TRUE)
+  expect_warning(data <- cdisc_data(adsl, adtte, adrs, check = TRUE), "deprecated")
```

Comment:
Just as @chlebowa said, this test is only successful once every 6 hours.

Using `lifecycle::expect_deprecated` works best to detect deprecated functions

```suggestion
  lifecycle::expect_deprecated(data <- cdisc_data(adsl, adtte, adrs, check = TRUE))
```

Reply:
Oooh, that's what it was!